parameter_files/Constant.java

```diff
@@ -0,0 +1,16 @@
+package parameter_files;
```

Comment:
Is it normal to have a class just for constants? I haven't seen this kind of style before